angular-chart.js

```diff
@@ -362,5 +365,11 @@
       var options = angular.extend({}, Chart.defaults.global, ChartJs.getOptions(type), scope.options);
       return options.responsive;
     }
+
+    function destroyChart(chart, scope) {
+      if(!chart) return;
```

Comment:
Nit: space between `!` and `chart` (I need to update the style checker for that).